pontoon/base/static/css/download_selector.css

```diff
@@ -0,0 +1,48 @@
+.download-selector {
+  float: right;
+}
+  
+.download-selector .selector {
+  padding-right: 0;
+  padding-top: 0;
+}
+  
+.download-selector .selector .fa {
+  float: right;
+  background: #3f4752;
+  border-radius: 3px;
+  box-sizing: border-box;
+  color: #AAA;
+  font-size: 16px;
+  height: 32px;
+  width: 32px;
+  padding-top: 8px;
+  text-align: center;
+}
+
+.download-selector .button:hover {
```

Comment:
My bad. This change breaks the behavior of hover. We should detect the hover on the `.fa` element.

So please change this line to: `.download-selector .selector .fa:hover {`.